src/flows/pipes/Notification/Expressions.tsx

```diff
@@ -0,0 +1,195 @@
+import React, {FC, useState, useMemo, useCallback, useContext} from 'react'
+
+import {
+  Input,
+  InputType,
+  IconFont,
+  EmptyState,
+  ComponentSize,
+} from '@influxdata/clockface'
+import {FromFluxResult} from '@influxdata/giraffe'
+import Expression from 'src/flows/pipes/Notification/Expression'
+import {FlowContext} from 'src/flows/context/flow.current'
+
+interface Props {
+  parsed: FromFluxResult
+  id: string
+}
+
+interface SchemaExpressions {
+  [key: string]: Array<string>
+}
+
+const Expressions: FC<Props> = ({id, parsed}) => {
+  const [search, setSearch] = useState('')
+  const {flow, updateData} = useContext(FlowContext)
+  const updateSearch = useCallback(
+    e => {
+      setSearch(e.target.value)
+    },
+    [search, setSearch]
+  )
+
+  const parsedResultToSchema = (parsed: FromFluxResult): SchemaExpressions => {
```

Comment:
seems like we can also move some of these functions outside of the component to reduce the overhead of re-declaring them every render function